app/models/comment.rb

```diff
@@ -1,10 +0,0 @@
-class Comment < ApplicationRecord
-  belongs_to :author, class_name: 'User'
-  belongs_to :post
-  after_save :comment_update_counter
-
-  def comment_update_counter
-    post = Post.find(post_id)
-    post.update_attribute(:comments_count, post.comments_count + 1)
-  end
-end
```

Comment:
- [ ] I think you have deleted all the association and model work that was needed in this milestone and your application. Please fix this.